supervisor/mounts/const.py

```diff
@@ -11,6 +11,7 @@
 ATTR_SERVER = "server"
 ATTR_SHARE = "share"
 ATTR_USAGE = "usage"
+ATTR_CIFS_VERSION = "cifs_version"
```

Comment:
Yea I think it should just be version. We can still validate it differently for NFS vs. CIFS if NFS version every became a thing we needed to worry about since we can differentiate on type. And it reads cleaner that way.